vanilla-framework/vanilla-framework
# Implementation of the new vertical rhythm

## Done
Converted the vertical rhythm from being based on margin-bottom to margin-top. Updated some spacing and font sizes to match the design.

## QA
- Pull down this branch and run `./run serve`
- Go to http://0.0.0.0:8101/vanilla-framework/examples/base/typo-scale/
- Check that the spacing and font sizes match [the design](https://github.com/vanilla-framework/vanilla-framework/issues/960#issue-214417670)

_Once this branch has landed. I will follow up with the removal of the typo-scale page from examples._

## Details
Fixes: https://github.com/vanilla-framework/vanilla-framework/issues/960
